app-tracking-protection/vpn-impl/src/main/res/values/donottranslate.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright (c) 2023 DuckDuckGo
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<resources>
+
+    <!-- Notify Me View on Downloads-->
+    <string name="appTrackingProtectionNotifyMeTitle" translatable="false">Keep an eye on blocked trackers</string>
```

Comment:
Why is this here? it should be moved to `donottranslate.xml` in `common-ui`. There's already Notify Me strings there.

Reply:
All your logic / layouts are in the `common-ui` module, why do you want to treat text differently?

Reply:
`common-ui` defines the logic / layouts and the modules using the components customise them. Those modules are the ones who can decide what strings to display using the component's custom attributes.

The strings from `donottranslate.xml` in `common-ui` are used for UI elements in components which are not customisable. One example is the label for the `Notify me` button which I added in `common-ui`.